card/gem_mods/ace_editor/card-mod-ace_editor.gemspec

```diff
@@ -0,0 +1,21 @@
+# -*- encoding : utf-8 -*-
+
+Gem::Specification.new do |s|
+  s.name = "card-mod-ace_editor"
+  s.version = "0.1"
```

Comment:
As long as this is in the same repo (and, more importantly, the same testing environment) as decko and card, I think we should probably make these versions use the same version as decko.  Eg `decko 0.10.1` must use `card-mod-edit 0.10.1`.  That's the easiest way to make sure they'll work together.  You can see what I'm talking about in decko.gemspec (although the card gem requires special handling because it predates decko and is already way past 1.0)

Reply:
Ok, I think I can just pull in the card version same as card.gemspec

Reply:
Oh, this one. I will see if this is easy(ish)

Reply:
I hope that is right. I locked the versions together in the card.gemspec which means they all release together.